src/GraphQL/FieldHelper/DataObjectFieldHelper.php

```diff
@@ -206,7 +207,7 @@ public function supportsGraphQL(Data $fieldDefinition, string $operationType)
             case 'mutation':
                 return $this->getGraphQlService()->supportsDataObjectMutationDataType($typeName);
             default:
-                throw new \Exception("unknown operation type");
+                throw new ClientSafeException("unknown operation type " . $typeName);
```

Comment:
Not sure about this one (and probably a few others) as this indicates a coding error. What do you think?

Reply:
Yes, you are right this exception should never happen in a normal case, only while coding new mutations/queries. However when it does happen it is no problem to show it to the client, so I don't see a good argument to change it back to a normal exception.

Reply:
Mhm, it will be still shown if pimcore is in debug mode